### ISSUE TITLE
Add a little bit of margin/padding to the items that are places close to the left and top edges of the screen to improve the scanning experience.

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#955]
For any interactive element that is close to the left edge of the screen, add just a few millimetres >5  to the left. With this extra space, someone who is using a switch interface will have just right amount of time to let go of their switch and press their switch.

Add the same amount of extra margin/padding to the top of the items that are close to the top of the screen for the same reason, such as the Quick exit button.

**Comment by cherylhjli** [comment]
For left/right margins, try 11rems (176px) (this is testing it out in the figma mockup, which is set at 1400px width with 12 columns and 30px gutters.

**Comment by cherylhjli** [comment]
Increase the height of the support bar to 80px. This should give a tiny bit more padding. I'm having trouble increasing more padding without making the support bar really big and heavy.

**Comment by greatislander** [comment]
See #1333

**Comment by jobara** [comment]
@greatislander did this get resolved with #1333? If not, what is the relationship?

**Comment by greatislander** [comment]
Ugh no it did not. I meant to address this as part of that PR but I didn't.

The padding on the `.center` utility classes should be adjusted as needed for this issue.
